scripts/release/release-changelog.sh

```diff
@@ -27,7 +27,7 @@ ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
 function usage() {
   [[ -n "${1}" ]] && echo "${1}"
   cat <<EOF
-usage: ${BASH_SOURCE[0]} -s <commit sha> -l <last release sha or tag> -d <directory>
+usage: ${BASH_SOURCE[0]} -s <commit sha> -l <last release sha or tag> -d <directory> [-n <current version number>]
```

Comment:
How do we use this flag?   is this only used for patch release?  
Did you update the release doc?  if so, I can read the doc first before I review this pr

Reply:
Yes please.